boards/nxp/frdm_mcxn236/doc/index.rst

```diff
@@ -0,0 +1,211 @@
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | soc flash                           |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+
+Targets available
+==================
+
+The default configuration file
+:zephyr_file:`boards/nxp/frdm_mcxn236/frdm_mcxn236_defconfig`
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The MCX-N236 SoC has 6 gpio controllers and has pinmux registers which
+can be used to configure the functionality of a pin.
+
++------------+-----------------+----------------------------+
+| Name       | Function        | Usage                      |
++============+=================+============================+
+| P0_PIO1_8  | UART            | UART RX                    |
++------------+-----------------+----------------------------+
+| P1_PIO1_9  | UART            | UART TX                    |
++------------+-----------------+----------------------------+
+
+System Clock
+============
+
+The MCX-N236 SoC is configured to use PLL0 running at 150MHz as a source for
+the system clock.
+
+Serial Port
+===========
+
+The FRDM-MCXN236 SoC has 8 FLEXCOMM interfaces for serial communication.
+Flexcomm 4 is configured as UART for the console.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the MCU-Link CMSIS-DAP Onboard Debug Probe.
+
+Using LinkServer
+----------------
+
+Linkserver is the default runner for this board, and supports the factory
+default MCU-Link firmware. Follow the instructions in
+:ref:`mcu-link-cmsis-onboard-debug-probe` to reprogram the default MCU-Link
+firmware. This only needs to be done if the default onboard debug circuit
+firmware was changed. To put the board in ``DFU mode`` to program the firmware,
+short jumper JP5.
+
+Using J-Link
+------------
+
+There are two options. The onboard debug circuit can be updated with Segger
+J-Link firmware by following the instructions in
+:ref:`mcu-link-jlink-onboard-debug-probe`.
+To be able to program the firmware, you need to put the board in ``DFU mode``
+by shortening the jumper JP5.
+The second option is to attach a :ref:`jlink-external-debug-probe` to the
+10-pin SWD connector (J12) of the board. Additionally, the jumper JP7 must
+be shortened.
+For both options use the ``-r jlink`` option with west to use the jlink runner.
+
+.. code-block:: console
+
+   west flash -r jlink
+
+Configuring a Console
+=====================
+
+Connect a USB cable from your PC to J17, and use the serial terminal of your choice
```

Comment:
```suggestion
Connect a USB cable from your PC to J10, and use the serial terminal of your choice
```

It seems like this is the USB connector used for the debugger, right?

![image](https://github.com/zephyrproject-rtos/zephyr/assets/34665051/122e5228-dd97-4bbb-84da-4ec16bdf1ed8)

Reply:
Updated.